build.gradle

```diff
@@ -4,18 +4,21 @@ buildscript {
     }
     repositories {
         mavenCentral()
+        jcenter()
```

Comment:
We need that repo because of mavenCentral does't contain asciidoctor-gradle-plugin.
I can replace jcenter with "plugins.gradle.org" repo.